File: vplanapp/src/main/java/xyz/jbapps/vplan/util/ParseException.java
package xyz.jbapps.vplan.util;

/**
 * @author Jannik Bamberger (dev.jbamberger@gmail.com)
 */

public class ParseException extends Exception {
    public ParseException() {
        super();
    }

    public ParseException(String message) {
        super(message);
    }

    public ParseException(String message, Throwable cause) {
        super(message, cause);
    }

    public ParseException(Throwable cause) {
        super(cause);
    }
}

File: fhgapp/src/main/java/de/jbamberger/fhgapp/repository/api/ParseException.kt
package de.jbamberger.fhgapp.repository.api

import java.io.IOException

/**
 * @author Jannik Bamberger (dev.jbamberger@gmail.com)
 */

internal class ParseException : IOException {
    constructor() : super()

    constructor(message: String) : super(message)

    constructor(message: String, cause: Throwable) : super(message, cause)

    constructor(cause: Throwable) : super(cause)
}
